core/chains/evm/config/config.go

```diff
@@ -6,7 +6,9 @@ import (
 
 	gethcommon "github.com/ethereum/go-ethereum/common"
 
-	"github.com/smartcontractkit/chainlink/v2/core/assets"
+	relayassets "github.com/smartcontractkit/chainlink-relay/pkg/assets"
```

Comment:
relayassets seems ok for now.
But will be a headache once we rename chainlink-relay to chainlink-common

Reply:
Agreed, but once we rename the repo, we would have to change the import path as well so let's do a cleanup then.